design-system/packages/segmented-control/src/SegmentedControl.tsx

```diff
@@ -107,8 +114,26 @@ export const SegmentedControl = ({
   }, [animate, selectedIndex]);
 
   return (
-    <Box {...props}>
-      <Root fill={fill} size={size} ref={rootRef} width={width}>
+    <Box
+      css={css`
+        outline: 0;
+        box-sizing: border-box;
+      `}
+      {...props}
+    >
+      <Root
+        css={css`
+          border: 1px solid #e1e5e9;
+          &:focus-within {
```

Comment:
This will work for most browsers, exception being `opera` certain `android` mobile browsers and `ie11`. 
I'm fine to not have a work around for these atm, but happy to revisit this decision if need be.